pkgs/by-name/li/linearmouse/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "linearmouse";
+  version = "0.10.0";
+
+  src = fetchurl {
+    url = "https://github.com/linearmouse/linearmouse/releases/download/v${version}/LinearMouse.dmg";
+    hash = "sha256-Ic0dEQEN1e78H/M2jOg2bEEU+ZtZ6nN6ykMJ+db7afE=";
+  };
+
+  sourceRoot = ".";
+
+  unpackCmd = ''
+    mnt=$(TMPDIR=/tmp mktemp -d -t nix-XXXXXXXXXX)
+
+    function finish {
+      /usr/bin/hdiutil detach $mnt -force
+      rm -rf $mnt
+    }
+    trap finish EXIT
+
+    /usr/bin/hdiutil attach -nobrowse -mountpoint $mnt $src
+    cp -a $mnt/LinearMouse.app $PWD/
+  '';
```

Comment:
Isn't there a default hook with 7z now?

Reply:
@SuperSandro2000 i cannot find .app file while using 7z, i just got 4.arm64 and 4.x64 file. And if i use undmg i got `error: only HFS file systems are supported.`. What should i do here?